lib/msal-common/src/client/SilentFlowClient.ts

```diff
@@ -190,9 +190,17 @@ export class SilentFlowClient extends BaseClient {
             },
             correlationId
         );
-        this.logger.info(
-            `Token refresh is required due to cache outcome: ${cacheOutcome}`
-        );
+        switch (cacheOutcome) {
+            case CacheOutcome.NOT_APPLICABLE:
+                this.logger.info(
+                    `Token refresh not required due to cache outcome: ${cacheOutcome}`
+                );
+                break;
+            default:
+                this.logger.info(
+                    `Token refresh is required due to cache outcome: ${cacheOutcome}`
+                );
+        }
```

Comment:
Case switch feels like overkill here. I'd suggest if/else but I think we already have cache hit messages so we likely should just log on a miss
```suggestion
        if (cacheOutcome !== CacheOutcome.NOT_APPLICABLE) {
                this.logger.info(
                    `Token refresh is required due to cache outcome: ${cacheOutcome}`
                );
        }
```

Reply:
+1. Especially since we are just logging and not adding any new functionality.